lib/wellTyped.ml

```diff
@@ -2281,7 +2281,7 @@ module WRPD = struct
                   pure
                     (let@ packing_ft =
                        WLAT.welltyped
-                         (fun loc it -> WIT.check loc oarg_bt it)
+                         (fun _ it -> WIT.check oarg_loc oarg_bt it)
```

Comment:
And then here you wouldn't need the location passed in as a separate argument. Also consider adding a brief comment to explain what that location argument you're ignoring (with the wildcard `_`) is.